packages/live-status-gateway/api/schemas/packages.yaml

```diff
@@ -0,0 +1,63 @@
+title: Packages
+description: Packages schema for websocket subscriptions
+$defs:
+  packages:
+    type: object
+    properties:
+      event:
+        type: string
+        const: packages
+      rundownPlaylistId:
+        description: Unique id of the rundown playlist, or null if no playlist is active
+        oneOf:
+          - type: string
+          - type: 'null'
+      packages:
+        description: The Package statuses for this playlist
+        type: array
+        items:
+          $ref: '#/$defs/package'
+    required: [event, rundownPlaylistId, packages]
+    additionalProperties: false
+    examples:
+      - event: packages
+        rundownPlaylistId: 'OKAgZmZ0Buc99lE_2uPPSKVbMrQ_'
+        packages:
+          $ref: '#/$defs/package/examples'
+  package:
+    type: object
+    properties:
+      packageName:
+        description: Name of the package
+        oneOf:
+          - type: string
+          - type: 'null'
+      statusCode:
+        description: Status code
```

Comment:
I think, considering that this is a Public API, we need to expand on what this status code is. I also would say, that probably exposing the internal Sofie Enum here is potentially problematic? We have no way of managing any changes on the Sofie side here. So I think Live Status Gateway should be translating this internal Sofie enum into a string enum that is defined by this API?